torch/nn/modules/transformer.py

```diff
@@ -0,0 +1,290 @@
+import torch
+import copy
+from .. import functional as F
+from .module import Module
+from .activation import MultiheadAttention
+from .container import ModuleList
+import numpy as np
```

Comment:
Remove this once you've deleted the numpy usage. We don't have a hard dependency on numpy.